client/post-editor/editor-ground-control/test/index.jsx

```diff
@@ -33,17 +30,14 @@ var MOCK_SITE = {
 	options: {}
 };
 
-var expect = chai.expect;
-
-chai.use( sinonChai );
-
 describe( 'EditorGroundControl', function() {
+	let EditorGroundControl;
+
+	useMockery();
+	useFakeDom();
 	this.timeout( 10 * 1000 );
```

Comment:
>can we drop the explicit timeout?

On second thought, I decided to opt for converting the tests to using Enzyme shallow rendering, which should be must faster, and dropped the timeout.